apps/myjobs/config/cable.yml

```diff
@@ -0,0 +1,10 @@
+development:
+  adapter: async
+
+test:
+  adapter: async
+
+production:
+  adapter: redis
+  url: <%= ENV.fetch("REDIS_URL") { "redis://localhost:6379/1" } %>
+  channel_prefix: job_constructor_production
```

Comment:
I assume this config is only read if ActionCable is used? Or does this add to the load time? Perhaps we should remove this file.

Reply:
I removed the file.